netmaster/master/endpointGroup.go

```diff
@@ -67,10 +73,14 @@ func CreateEndpointGroup(tenantName, networkName, groupName string) error {
 	// params for docker network
 	if GetClusterMode() == "docker" {
 		// Create each EPG as a docker network
-		err = docknet.CreateDockNet(tenantName, networkName, groupName, nwCfg)
-		if err != nil {
-			log.Errorf("Error creating docker network for group %s.%s. Err: %v", networkName, groupName, err)
-			return err
+		if aciMode && nwCfg.PktTagType == "vxlan" {
+			log.Info("Skipping docker network creation because of ACI mode and nwtype = %v", nwCfg.PktTagType)
```

Comment:
@gaurav-dalvi: I think it is useful to allow creation of networks with vxlan encap even in aci mode. We used to allow this.